### PR TITLE
Adding Globe icon

### PR DESCRIPTION
**Pull request by gjones**
### Summary 
This PR adds a new `Globe` icon and updates the `Stop` icon due to feedback. The old stop icon is now `Square`.

CleanShot 2024-11-04 at 13 01 15@2x

CleanShot 2024-11-05 at 06 21 12@2x

### CODE REVIEW

File: src/components/icons/Globe.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M21 12C21 16.9706 16.9706 21 12 21M21 12C21 7.02944 16.9706 3 12 3M21 12H3M12 21C7.02944 21 3 16.9706 3 12M12 21C14.2512 18.5355 15.5305 15.3372 15.6 12C15.5305 8.66283 14.2512 5.46452 12 3M12 21C9.74885 18.5355 8.46952 15.3372 8.4 12C8.46952 8.66283 9.74885 5.46452 12 3M3 12C3 7.02944 7.02944 3 12 3"
+      stroke="white"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
```
Comment (serdec) [inline src/components/icons/Globe.tsx:17]: this-case will generate errors in the ui, change to camelCase

File: src/components/Icon/IconCommon.tsx [hunk 2]
```diff
@@ -206,6 +207,7 @@ export const ICONS_MAP = {
   "folder-open": FolderOpen,
   gear: Gear,
   gift: Gift,
+  globe: Globe,
```
Comment (serdec) [inline src/components/Icon/IconCommon.tsx:210]: you'll need to add it here too https://github.com/ClickHouse/click-ui/blob/739d7f86e60adfa7793f64440d6a214b81a464b7/src/components/Icon/types.ts#L10

File: src/components/icons/Globe.tsx (hunk 1 above)
Comment (gjones) [reply src/components/icons/Globe.tsx]: one day ill remember

File: src/components/Icon/IconCommon.tsx [hunk 3]
```diff
@@ -91,6 +92,7 @@ import SlideOut from "@/components/icons/SlideOut";
 import Sparkle from "@/components/icons/Sparkle";
 import Speaker from "@/components/icons/Speaker";
 import Speed from "@/components/icons/Speed";
+import Square from "../icons/Square";
```
Comment (serdec) [inline src/components/Icon/IconCommon.tsx:95]: let's use the absolute import as above

**Review by serdec — APPROVED** [review]

**Pull request merged**